layer.md

```diff
@@ -1,63 +1,109 @@
-# Creating an Image Filesystem Changeset
+# Filesystem Layer Changeset
 
-An example of creating an Image Filesystem Changeset follows.
+A layer changeset is the distributable difference between filesystems.
+A layer changeset is not directly extracted, but applied to a filesystem.
 
-An image root filesystem is first created as an empty directory.
-Here is the initial empty directory structure for a changeset using the randomly-generated directory name `c3167915dc9d` ([actual layer DiffIDs are generated based on the content](#id_desc)).
+## Distributable Format
+
+Layer Changesets for the [mediatype](./media-types.md) `application/vnd.oci.image.layer.tar+gzip` MUST be packaged in [tar archive][tar-archive].
+Layer Changesets for the [mediatype](./media-types.md) `application/vnd.oci.image.layer.tar+gzip` MUST NOT include duplicate entries for file paths in the resulting [tar archive][tar-archive].
+
+## Change Types
+
+Types of changes that can occur in a changeset are:
+
+* Added
+* Modified
+* Removed
+
+Additions and Modifications are reflected the same in the changeset tar archive.
+
+Removals are reflected using "[whiteout](#whiteouts)" file entries (See [Representing Changes](#representing-changes)).
+
+## Creating
+
+### Initial Root Filesystem
+
+An image root filesystem has an initial state as an empty directory.
+The name of this not relevant to the layer itself, only for the purpose of producing comparisons.
+
+Here is an initial empty directory structure for a changeset, with a unique directory name `rootfs-c9d-v1`.
 
 ```
-c3167915dc9d/
+rootfs-c9d-v1/
 ```
 
+### Populate Initial Filesystem
+
 Files and directories are then created:
 
 ```
-c3167915dc9d/
+rootfs-c9d-v1/
     etc/
         my-app-config
     bin/
         my-app-binary
         my-app-tools
 ```
 
-The `c3167915dc9d` directory is then committed as a plain Tar archive with entries for the following files:
+The `rootfs-c9d-v1` directory is then committed as a plain [Tar archive][tar-archive] with relative path to `rootfs-c9d-v1`.
+Entries for the following files:
 
 ```
-etc/my-app-config
-bin/my-app-binary
-bin/my-app-tools
+./
+./etc/
+./etc/my-app-config
+./bin/
+./bin/my-app-binary
+./bin/my-app-tools
 ```
 
-To make changes to the filesystem of this container image, create a new directory, such as `f60c56784b83`, and initialize it with a snapshot of the parent image's root filesystem, so that the directory is identical to that of `c3167915dc9d`.
-NOTE: a copy-on-write or union filesystem can make this very efficient:
+### Populate a Comparison Filesystem
+
+Create a new directory and initialize it with an identical copy or snapshot of the prior root filesystem.
```

Comment:
I wondered about that. Something like `cp -pav <src> <dst>` or `rsync -avPHS <src> <dst>` or so on. These are all very Linux centric, which may be fine for starters, but this line is more of the rule, rather than the example.
Perhaps, having a statement of the files, their contents, and all available attributes (timestamps, mode, owner, group, xattrs, etc)?